changelog/next/features/3899--async-policy.md

```diff
@@ -0,0 +1,3 @@
+We introduced a new `tenzir.log-overflow-policy` settings key which
+allows the user to control the behavior when the log queue runs full.
+Accepted values are `block` or `overrun_oldest`.
```

Comment:
This should mention the default value, especially if we want to change it from the old default of `block`